index.js

```diff
@@ -35,36 +37,80 @@ module.exports = {
   included: function(app) {
     patchEmberApp(app);
 
-    var mode;
-
-    if (isFastBoot()) {
-      process.env[APP_NAME_KEY] = app.name;
-      app.options.autoRun = false;
+    // We serve the index.html from fastboot-dist, so this has to apply to both builds
+    app.options.storeConfigInMeta = false;
```

Comment:
Is it because the server doesn't read th index.html, so when the app.js is evaluated for the server, the config will be missin?